### PR TITLE
fix indentation and numbering

### PR DESCRIPTION
**Pull request by keegoid-nr**
- It might make sense to remove some of the itemized lists for clarity.
- fix for log lines to check in SJM
cc @jeff-colucci

### CODE REVIEW

File: src/content/docs/synthetics/synthetic-monitoring/private-locations/job-manager-configuration.mdx [hunk 1]
```diff
@@ -1099,35 +1102,36 @@ To set permanent data storage on Docker:
...
-Example:
 
-```
-		docker run ... -v /sjm-volume:/var/lib/newrelic/synthetics:rw ...
-```
+    Example:
+
+    ```sh
+    docker run ... -v /sjm-volume:/var/lib/newrelic/synthetics:rw ...
+    ```
```
Comment (clarkmcadoo) [inline src/content/docs/synthetics/synthetic-monitoring/private-locations/job-manager-configuration.mdx:1111]:
```suggestion
Example:

```sh
  docker run ... -v /sjm-volume:/var/lib/newrelic/synthetics:rw ...
```
```

Comment (keegoid-nr) [reply src/content/docs/synthetics/synthetic-monitoring/private-locations/job-manager-configuration.mdx]: Granted, I have not previewed the change in VS Code (I still need to learn how to do that), but the goal here is to nest the Example under the itemized list. This becomes especially obvious when not nesting breaks the counter of the itemized list. Example:
image

**Review by jeff-colucci — APPROVED** [review]

**Comment by keegoid-nr** [comment]
@jeff-colucci This should be one code block. The indentation is breaking the code block.
image

**Comment by keegoid-nr** [comment]
If we're going to use itemized lists, it should start with 1 at the first step.
image

**Comment by jeff-colucci** [comment]
@keegoid-nr Ah ok, let me take a crack at fixing this in a PR myself, I'll have it forwarded to you today or tomorrow, but it will be done this week.

**Pull request merged**